app.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+const app = express();
+const usersList = require("./userService");
+
+app.use(express.static('public'));
+
+app.get("/", function(req, res) {
+    res.render("index.pug");
+});
+
+// Load page
+app.get("/usersCenter", function(req, res) {
```

Comment:
https://ux.stackexchange.com/questions/41595/what-is-the-casing-convention-for-url-routes

Reply:
Fixed.